automated install/basic-install.sh

```diff
@@ -2489,6 +2522,9 @@ main() {
     # but before starting or resttarting the ftl service
     disable_resolved_stublistener
 
+    # Disable lighttpd server
+    disableLighttpd
```

Comment:
This is placed here unconditionally. It will trigger on each update/checkout. 
We should put it into the migration part (where it is guarded by `if [[ -f "${PI_HOLE_V6_CONFIG}" ]]; then` and additionally check for the presence and enable/disable state of `lighttpd` at all.

Reply:
https://github.com/pi-hole/pi-hole/pull/5849